Remove rook operator from docs in all versions (#13114)
yugabyte/yugabyte-db: docs/content/preview/deploy/kubernetes/single-zone/oss/yugabyte-operator.md

      Operator Hub
    </a>
  </li>
</ul>

> Create and manage a YugabyteDB cluster with a Kubernetes native custom resource `ybcluster.yugabyte.com`. The custom resource definition and other necessary specifications can be found in [YugabyteDB operator repository](https://github.com/yugabyte/yugabyte-operator/). See full list of configuration flags [here](#configuration-flags).

Reviewer: ```suggestion
Create and manage a YugabyteDB cluster with a Kubernetes native custom resource `ybcluster.yugabyte.com`. The custom resource definition and other necessary specifications are available in [YugabyteDB operator repository](https://github.com/yugabyte/yugabyte-operator/). In addition, you may consult the full list of [configuration flags](#configuration-flags).
```

Author: Fixed in https://github.com/yugabyte/yugabyte-db/pull/13114/commits/1d770cab431167ca7fe5bb0c798bdbe50929cafb
